scout/parse/variant/callers.py

```diff
@@ -28,36 +36,51 @@ def parse_callers(variant, category="snv"):
     svdb_origin = variant.INFO.get("svdb_origin")
     raw_info = variant.INFO.get("set")
 
+    filter_status_default = "Pass"
+    if variant.FILTER is not None:
+        filter_status_default = "Filtered - {}".format(variant.FILTER.replace(";", " - "))
+
     if other_info:
-        for info in other_info.split(","):
+        infos = other_info.split(",")
+        if len(infos) > 1:
```

Comment:
As per the doctoring: add filter status if we are looking at a single caller, else just set as before.